docs/new-generator.md

```diff
@@ -211,27 +211,25 @@ No model defined in this package
 {{/modelPackage}}
 {{/generateModelDocs}}
 
-<a name="documentation-for-authorization"></a>{{! TODO: optional documentation for authorization? }}
+<a id="documentation-for-authorization"></a>{{! TODO: optional documentation for authorization? }}
 ## Documentation for Authorization
 
-{{^authMethods}}
-All endpoints do not require authorization.
-{{/authMethods}}
-{{#authMethods}}
-{{#last}}
-Authentication schemes defined for the API:
-{{/last}}
-{{/authMethods}}
+{{^authMethods}}Endpoints do not require authorization.{{/authMethods}}
+{{#hasAuthMethods}}Authentication schemes defined for the API:{{/hasAuthMethods}}
 {{#authMethods}}
-<a name="{{name}}"></a>
+<a id="{{name}}"></a>
 ### {{name}}
```

Comment:
Question to reviewers of this PR: I saw {{{name}}} in the README of some generators: For authentication schemes, is it really likely to play a role? Which one should be the default for new generators?

Reply:
> For authentication schemes, is it really likely to play a role?

Depending on how you define playing a role. Users should be able to figure out what this authenticaiton/authoriazation does by reading the `name`

> Which one should be the default for new generators?

Sorry I don't understand the question.

Reply:
@wing328  the questions I asked are about the number of curly brackets to use: should it rather be `{{name}}` or `{{{name}}}`?
For authentication schemes, is the number of curly brackets in the expression really likely to be impact the resolved text? Which number of brackets in the expression for authentication names should be the default for new generators?